apps/web/public/static/locales/es/common.json

```diff
@@ -563,6 +563,7 @@
   "calendar_days": "Días del Calendário",
   "business_days": "Días Laborales",
   "set_address_place": "Establecer una dirección o un lugar",
+  "set_city_place": "Establecer una ciudad",
```

Comment:
You should only change the English lang internationalization, others are autogenerated by an external service (Crowdin). So I think this can be removed.